<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="shadow">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(45, 48, 55);
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1211</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-top-left-radius : 10px;
border-top-right-radius : 10px;
background-color: #E7B18E;</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>971</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Casta Thin</family>
          <pointsize>68</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: E7B18E;</string>
        </property>
        <property name="text">
         <string>Wrinkles Face Detection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_close">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>18</width>
          <height>18</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #FF605C;
border-radius: 9%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_minimize">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>18</width>
          <height>18</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #FFBD44;
border-radius: 9%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_gtw">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>18</width>
          <height>18</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #00CA4E;
border-radius: 9%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>260</y>
         <width>381</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid;
border-color: rgb(231, 177, 142);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>260</y>
         <width>381</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid;
border-color: rgb(231, 177, 142);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #E7B18E;</string>
       </property>
       <property name="text">
        <string>Input</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>170</y>
         <width>121</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #E7B18E;</string>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>610</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(45, 48, 55);
color: #E7B18E;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_loadImage">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>690</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #E7B18E;
border: 1px solid #E7B18E; 
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Load Image</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_saveImage">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>690</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #E7B18E;
border: 1px solid #E7B18E; 
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_process">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>430</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #E7B18E;
border: 1px solid #E7B18E; 
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Process</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
